src/components/ec-panel/ec-panel.vue

```diff
@@ -78,65 +107,93 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/index';
-@import '../../scss/tools/index';
+<style>
+.ec-panel {
+  max-width: 352px;
 
-$ec-panel-header-btn-fill: $level-4-interactive-elements !default;
-$ec-panel-header-btn-fill-hover: $level-4-tech-blue !default;
-$ec-panel-background-color: $white !default;
+  @apply tw-w-full;
+  @apply tw-absolute tw-right-0 tw-top-0;
 
-.ec-panel {
-  width: 100%;
-  max-width: $side-panel-max-width;
-  height: 100vh;
-  min-height: 100%;
-  position: absolute;
-  background: $ec-panel-background-color;
-  right: 0;
-  top: 0;
-  padding: 16px 24px 24px 24px;
-  overflow-y: auto;
-
-  @include scrollbar;
-  @include left-box-shadow-level-1;
+  &__content {
+    max-width: 352px;
 
-  &__header {
-    display: block;
+    @apply tw-w-full tw-h-screen;
+    @apply tw-fixed;
+    @apply tw-flex tw-flex-col;
+    @apply tw-bg-light;
+    @apply tw-shadow-level-1-rtl;
   }
 
   &__header-icons {
-    display: flex;
-    flex-direction: row;
+    @apply tw-flex tw-flex-row;
   }
 
   &__header-icon {
-    fill: currentColor;
-    display: inline-block;
-    vertical-align: top;
+    @apply tw-fill-current;
+    @apply inline-block;
+    @apply tw-align-top;
   }
 
   &__header-action {
-    cursor: pointer;
-    color: $ec-panel-header-btn-fill;
-
-    @include color-transition;
+    @apply tw-cursor-pointer;
+    @apply tw-text-gray-4;
+    @apply tw-transition-text tw-duration-300 tw-ease-out;
 
     &:hover {
-      color: $ec-panel-header-btn-fill-hover;
+      @apply tw-text-key-4;
     }
 
     &--back {
-      margin-right: auto;
+      @apply tw-mr-auto;
     }
 
     &--close {
-      margin-left: auto;
+      @apply tw-ml-auto;
     }
   }
+
+  &__header {
+    @apply tw-block;
+    @apply tw-mt-16 tw-mx-24;
+  }
+
+  &__main-container {
+    @apply tw-overflow-y-auto;
+    @apply tw-m-24;
+    @apply tw-pr-8;
+
+    @screen sm {
+      @apply tw-mr-8;
+    }
+
+    scrollbar-width: thin;
```

Comment:
We don't have an 8px px one. I added one in the mixins. After talk with design the md-scrollbar(we can rename) should be the default one. It also makes sense since 8 can be divided by 4. 10 and 6 respectively cannot.